e2e/test/scenarios/models/models-list-view.cy.spec.js

```diff
@@ -336,5 +334,63 @@ describe("scenarios > models list view", () => {
       H.undoToast().should("contain.text", "This is a question now");
       cy.findByTestId("list-view").should("not.exist");
     });
+
+    it("should consider mini bar chart setting for quantity/score columns", () => {
+      H.restore();
+      cy.signInAsAdmin();
+      cy.intercept("POST", "/api/dataset").as("dataset");
+
+      H.createNativeQuestion({
+        name: "Native Model",
+        type: "model",
+        native: {
+          query: "SELECT * FROM ORDERS LIMIT 5",
+        },
+      }).then(({ body: { id: nativeModelId } }) => {
+        cy.request("PUT", `/api/card/${nativeModelId}`, {
+          display: "list",
```

Comment:
We should be able to just pass `display: "list"` to `H.createNativeQuestion`, no need for 2 API requests.